cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -2,15 +2,23 @@ package cli
 
 import (
 	"fmt"
+	"io"
+	"os"
 	"time"
 
 	"github.com/alecthomas/kong"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
 	"github.com/square/spiffe-aws-assume-role/cmd/spiffe-aws-assume-role/cli/mappers"
 	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
 	"github.com/square/spiffe-aws-assume-role/pkg/processcreds"
 )
 
+const (
+	logFileName = "spiffe-aws-assume-role.log"
```

Comment:
Can we make this configurable through the CLI so we internally we can choose where to write the logs. Also, this might be better as a file path

Reply:
Will do